src/ceph-volume/ceph_volume/devices/raw/activate.py

```diff
@@ -35,21 +35,25 @@ def activate_bluestore(meta, tmpfs, systemd, block_wal=None, block_db=None):
     system.chown(osd_path)
     prime_command = [
         'ceph-bluestore-tool',
-        'prime-osd-dir', '--dev', meta['device'],
+        'prime-osd-dir',
         '--path', osd_path,
-        '--no-mon-config']
+        '--no-mon-config',
+        '--dev', meta['device'],
+    ]
     process.run(prime_command)
 
     # always re-do the symlink regardless if it exists, so that the block,
     # block.wal, and block.db devices that may have changed can be mapped
     # correctly every time
-    prepare_utils.link_block( meta['device'], osd_id)
+    prepare_utils.link_block(meta['device'], osd_id)
+
+    if 'device_db' in meta:
+        prepare_utils.link_db(meta['device_db'], osd_id, osd_uuid)
 
     if block_wal:
```

Comment:
Looks like `block_wal` is gone from the signature, where is it coming from now? Should we just remove it since we have `elif 'device_wal' in meta:`?